src/client/auth/abstractAuthentication.ts

```diff
@@ -19,6 +19,20 @@ export interface AuthenticationMethod {
 	): Promise<AuthenticationResult>
 }
 
+export interface AuthenticationMethodUN {
+	TYPE: string
+	getTokenProof(request: AuthenticateInterface): Promise<AuthenticationResult>
+}
+
+export interface MultiAuthenticateInterface {
+	options?: any
+}
+
+export interface AuthenticationMethodMulti {
+	TYPE: string
+	getTokenProofMulti(userTokens: Array<any>, request: MultiAuthenticateInterface): Promise<AuthenticationResult>
```

Comment:
@nicktaras interface describes type of "request", but implementation use "any", better use specific type, it can help avoid mistakes and IDE helps to code.

Reply:
Thanks, I must have missed this. I'll use the interface.

Reply:
could you create interface for userTokens ? 
thanks